lib/pbench/server/cache_manager.py

```diff
@@ -739,9 +605,7 @@ def full_discovery(self):
         self._discover_controllers()
 
     def __contains__(self, dataset_id: str) -> bool:
-        """
-        Allow asking whether a CacheManager contains an entry for a specific
-        dataset.
+        """Ask whether a CacheManager contains a specific dataset.
```

Comment:
Just a nit, but I think the _caller_ is "asking"; this function will "answer" or "report" or "indicate", etc.

Reply:
Gimme a break; the old version said "allow asking". This is an improvement. But "why would you call it" vs "what does it do" is always a question... 👀